ambari-admin/pom.xml

```diff
@@ -19,15 +19,15 @@
   <parent>
     <groupId>org.apache.ambari</groupId>
     <artifactId>ambari-project</artifactId>
-    <version>2.0.0.0-SNAPSHOT</version>
+    <version>2.7.0.0.0</version>
```

Comment:
Snapshot word shouldn't be removed. Likewise at other places

Reply:
Since this a new branch cut-off from trunk this got replaced when running the command 
mvn versions:set -DnewVersion=2.7.0.0.0

Reply:
Taking my comment back. SNAPSHOT should go off, as branch is created rom trunk